SPECS-EXTENDED/libgphoto2/libgphoto2.spec

```diff
@@ -3,19 +3,18 @@ Distribution:   Azure Linux
 %bcond_with gp2ddb
 
 %global udevdir %(pkg-config --variable=udevdir udev)
-%global port_version 0.12.0
+%global port_version 0.12.2
 
 Name:           libgphoto2
-Version:        2.5.27
-Release:        2%{?dist}
+Version:        2.5.31
+Release:        1%{?dist}
 Summary:        Library for accessing digital cameras
 # GPLV2+ for the main lib (due to exif.c) and most plugins, some plugins GPLv2
 License:        GPLv2+ and GPLv2
 URL:            http://www.gphoto.org/
 
 Source0:        http://downloads.sourceforge.net/gphoto/%{name}-%{version}.tar.bz2
```

Comment:
Downloading source from this URl is not happening. Can you please update the correct url here?